packages/tf-next/src/commands/delete-deployment.ts

```diff
@@ -0,0 +1,164 @@
+import { ApiGatewayV2, CloudWatchLogs, IAM, Lambda, S3 } from 'aws-sdk';
+import { query } from 'jsonpath';
+import * as path from 'path';
+
+const apiGatewayV2 = new ApiGatewayV2();
+const cloudWatch = new CloudWatchLogs();
+const iam = new IAM();
+const lambda = new Lambda();
+const s3 = new S3();
+
+type LogLevel = 'verbose' | 'none' | undefined;
+
+interface DeleteDeploymentProps {
```

Comment:
Aside: Instead of Props I tend toward "Params" term but maybe this repo has this convention. I know React goes with Props.

Reply:
Yeah, this is based on the code that existed before.